### PR TITLE
Added ability to delete profile picture

### PR DESCRIPTION
**Pull request by nrs1729**
#### Describe the changes you have made in this PR: 
Added a button on the show profile page to delete the profile picture. Issue #595 

### Screenshots of the changes (If any):
image

### CODE REVIEW

**Comment by nrs1729** [comment]
Oh sorry forgot about tests, I'll add them

File: app/controllers/users/logix_controller.rb [hunk 1]
```diff
@@ -17,6 +17,12 @@ def profile
     @profile = ProfileDecorator.new(@user)
   end
 
+  def delete_profile_picture
```
Comment (ankitkataria) [inline app/controllers/users/logix_controller.rb]: Could we use the `update` route for users itself for this?

File: spec/controllers/users_logix_controller_spec.rb [hunk 2]
```diff
@@ -63,4 +63,12 @@
     expect(@user.country).to eq("IN")
     expect(@user.educational_institute).to eq("MAIT")
   end
+  it "should delete user profile picture" do
+    post delete_profile_picture_path(@user)
+    expect(response).to redirect_to(profile_path)
+    expect(@user.profile_picture.exists?).to eq(false)
+  end
 end
+#it "profile picture should exist in test user object before deleted" do
```
Comment (ankitkataria) [inline spec/controllers/users_logix_controller_spec.rb]: Let's remove this if not required.

**Comment by nitin10s** [comment]
@satu0king Old UI. Please close

**Pull request closed without merging**